modules/management-console-agent/src/test/java/org/apache/ignite/internal/agent/dto/topology/TopologySnapshotTest.java

```diff
@@ -28,6 +27,7 @@
 import org.apache.ignite.spi.discovery.tcp.internal.TcpDiscoveryNode;
 import org.junit.Test;
 
+import static org.apache.ignite.internal.util.lang.GridFunc.asList;
```

Comment:
May be it will be more natural to "import F" ?
And use in code `F.asList(....)` ?